persistence-model/src/main/resources/application.conf

```diff
@@ -77,6 +77,10 @@ cassandra-journal {
   connect-retries = 10
   connect-retry-delay = 5s
   reconnect-max-delay = 60s
+  authentication.username = ""
```

Comment:
The code never appears to refer to these `cassandra-journal.authentication.username` and `cassandra-snapshot-store.authentication.username`. Should it?

(I see that the code _does_ refer to the settings in the query project's `application.conf`, so I'm not worried about those)